src/game/g_cmds.cpp

```diff
@@ -4540,14 +4544,18 @@ void Cmd_Call_f(gentity_t *ent) {
     }
   }
 
+  Utilities::copyStanceFromClient(other, ent);
```

Comment:
These bits of `call` and `goto` code look identical, could maybe move to a common function such as `void MovePlayerToOther(gentity_t *player, gentity_t *other)`